packages/fuel-indexer-lib/src/graphql/parser.rs

```diff
@@ -633,6 +648,19 @@ impl ParsedGraphQLSchema {
         }
     }
 
+    /// Return the GraphQL type for a given entity.
+    fn entity_type(&self, name: &str) -> Option<&String> {
+        self.entity_names_to_types.get(name)
+    }
+
+    /// Return the GraphQL type for a given entity or field.
```

Comment:
```suggestion
    /// Return the GraphQL type for a given `TypeDefinition` name or `FieldDefinition` name.
```
- I think there should also be a little comment here explaining why we use this in `ParsedGraphQLSchema` as opposed to letting the caller decide to call `ParsedGraphQLSchema::field_type` then `ParsedGraphQLSchema::entity_type` itself. If the answer is "convenience" that's totally fine

Reply:
It's just convenience. 👍🏽 `QUERY_ROOT` is essentially communicated using a `None` value for `field_type` in `Selection::new()` now, so I decided to hide that detail away so that all we have to do is just pass `field_type` into `ParsedGraphQLSchema::graphql_type()` to get the correct value.